action.yml

```diff
@@ -53,20 +53,50 @@ inputs:
 runs:
   using: 'composite'
   steps:
+    - name: Check if review is requested
+      shell: bash
+      env:
+        PR_TITLE: ${{ github.event.pull_request.title }}
+        PATTERN: \\b((?:no|skip)-(?:review|otter|otterai)|otter-(?:no|bye|restricted))(?:,((?:no|skip)-(?:review|otter|otterai)|otter-(?:no|bye|restricted)))*\\b
```

Comment:
🔍 Ensure the regex pattern covers all necessary cases for skipping reviews. Review the conditions to avoid false negatives.